bundles/pom.xml

```diff
@@ -178,6 +178,7 @@
     <module>org.openhab.binding.mail</module>
     <module>org.openhab.binding.max</module>
     <module>org.openhab.binding.mcp23017</module>
+    <module>org.openhab.binding.mecmeter</module>
```

Comment:
It also needs to be added to the BOM pom.xml.

Reply:
Thought I did, but you are right. Added it.